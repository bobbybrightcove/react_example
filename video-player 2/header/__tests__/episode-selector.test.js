import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeSelector from '../episode-selector';
import contentItem from '../../__tests__/content-item.json';


describe('EpisodeSelector', () => {
  const player = {};
  const episode = contentItem;

  it('renders according to valid snapshot', () => {
    const component = renderer.create(
      <EpisodeSelector
        player={player}
        episode={episode}
      />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
