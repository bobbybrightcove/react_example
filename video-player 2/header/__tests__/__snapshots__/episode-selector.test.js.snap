// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodeSelector renders according to valid snapshot 1`] = `
<div
  className="episodeSelectorWrapper"
>
  <div
    className="episodeSelectorBox"
    onClick={[Function]}
  >
    <div
      className="episodeSelectorText"
    >
      S1.E688
      <img
        alt="Episodes"
        src="yellow-arrow-down.svg"
      />
    </div>
  </div>
</div>
`;
